cogs/SquadQueue.py

```diff
@@ -539,14 +635,29 @@ async def ping_staff_autocomplete(self, interaction: discord.Interaction, curren
             for role_name in self.helper_staff_roles if current.lower() in role_name.lower()
         ]
 
+    def allow_new_mogis(self):
+        self.LAUNCH_NEW_EVENTS = True
+
     @app_commands.command(name="annul_current_mogi")
     @app_commands.guild_only()
-    async def annul_current_mogi(self, interaction: discord.Interaction):
-        """The mogi currently gathering will be deleted.  The queue resumes at the next hour.  Staff use only."""
+    async def annul_current_mogi(self, interaction: discord.Interaction, resume_mogis_after: bool):
+        """The mogi currently gathering will be deleted.  resume_mogis_after determines whether future mogis will be scheduled.  Staff use only."""
         self.next_event = None
         self.ongoing_event = None
+        self.LAUNCH_NEW_EVENTS = False
+        msg = "The current mogi has been cancelled, "
+        if resume_mogis_after:
+            next_event_open_time = self.compute_next_event_open_time()
+            next_event_start_time = (
+                next_event_open_time + self.QUEUE_OPEN_TIME)
+            delay = (next_event_start_time -
+                     datetime.now(timezone.utc)).total_seconds()
+            asyncio.get_event_loop().call_later(delay, self.allow_new_mogis)
+            msg += f"the queue will resume at {next_event_start_time}."
+        else:
+            msg += "future mogis will not be started."
```

Comment:
There are a few potential bugs if staff attempts a more complex usage of this command. For example, annulling, resuming, and annulling again with a later time than the first would result in unexpected behavior (the 1st would open the queue).

Reply:
i considered that but i couldnt really think of another way to do it, and i dont know how to cancel an aynscio function call thats been queued up.  I'm kinda just hoping they dont fuck it up, the effects arent that bad and its a command not available to the general public.

Reply:
I don’t think a fix is terribly high priority, but you could have a variable in SquadQueue:
`self.annul_time_handler = None`

line 655:
`self.annul_time_handler = asyncio.get_event_loop().call_later(delay, self.allow_new_mogis)`

then between line 647 and 648:
`if self.annul_time_handler is not None:
    self.annul_time_handler.cancel()`